src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -5,6 +5,18 @@
 *REMOVED*Shared Microsoft.CodeAnalysis.VisualBasic.VisualBasicSyntaxTree.ParseText(text As String, options As Microsoft.CodeAnalysis.VisualBasic.VisualBasicParseOptions = Nothing, path As String = "", encoding As System.Text.Encoding = Nothing, cancellationToken As System.Threading.CancellationToken = Nothing) -> Microsoft.CodeAnalysis.SyntaxTree
 *REMOVED*Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.ParseSyntaxTree(text As Microsoft.CodeAnalysis.Text.SourceText, options As Microsoft.CodeAnalysis.ParseOptions = Nothing, path As String = "", cancellationToken As System.Threading.CancellationToken = Nothing) -> Microsoft.CodeAnalysis.SyntaxTree
 *REMOVED*Shared Microsoft.CodeAnalysis.VisualBasic.SyntaxFactory.ParseSyntaxTree(text As String, options As Microsoft.CodeAnalysis.ParseOptions = Nothing, path As String = "", encoding As System.Text.Encoding = Nothing, cancellationToken As System.Threading.CancellationToken = Nothing) -> Microsoft.CodeAnalysis.SyntaxTree
+Microsoft.CodeAnalysis.VisualBasic.AggregateClauseSymbolInfo.Select1() -> Microsoft.CodeAnalysis.SymbolInfo
```

Comment:
I presume these are all coming from @CyrusNajmabadi's fix to the public API analyzer to correctly track auto-properties in VB? Should these entries be moved to Shipped API txt?

Reply:
Yes, that's the source of the difference. Can you verify that all relevant additions were already shipped?